Assets/Scripts/Player.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Player : MonoBehaviour
+{
+    public Rigidbody2D body;
+    public SpriteRenderer sprite;
+    public Animator animator;
+    public float speed = 8.0f;
```

Comment:
A good practice in Unity is to let the members in `private` access and add the `[SerializeField]` attribute to the lines of those you want to assign in the Unity Inspector. Encapsulation is guaranteed this way.